VocaDb.ResXFileCodeGenerator.Tests/GeneratorTests.cs

```diff
@@ -139,198 +145,248 @@ namespace Resources
     using System.Globalization;
     using System.Resources;
 
-    {(publicClass ? "public" : "internal")} static class ActivityEntrySortRuleNames
+    {(publicClass ? "public" : "internal")}{(partial ? " partial" : "")}{(staticClass ? " static" : "")} class ActivityEntrySortRuleNames
     {{
-        private static ResourceManager? s_resourceManager;
-        public static ResourceManager ResourceManager => s_resourceManager ??= new ResourceManager(""VocaDb.Web.App_GlobalResources.ActivityEntrySortRuleNames"", typeof(ActivityEntrySortRuleNames).Assembly);
-        public static CultureInfo? CultureInfo {{ get; set; }}
-
-        /// <summary>
-        /// Looks up a localized string similar to Oldest.
-        /// </summary>
-        public static string CreateDate => ResourceManager.GetString(nameof(CreateDate), CultureInfo)!;
-
-        /// <summary>
-        /// Looks up a localized string similar to Newest.
-        /// </summary>
-        public static string CreateDateDescending => ResourceManager.GetString(nameof(CreateDateDescending), CultureInfo)!;
+        {(publicClass ? "public" : "internal")}{(partial ? " partial" : "")}{(staticClass ? " static" : "")} class {innerClassName}
+        {{{(string.IsNullOrEmpty(innerClassInstanceName) ? "" : $"\n            public {innerClassInstanceName} {{ get; }} = new();\n")}
```

Comment:
The type name is missing, which causes failing tests. By the way, is it the intended behavior that this property is generated for the "inner" class? Should it be generated for the container class instead as you mentioned in #5? I know you're still editing...

Reply:
Indeed, I made a total mess of that